contracts/ControlledGateway.sol

```diff
@@ -0,0 +1,34 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Pausable} from "@openzeppelin/contracts/security/Pausable.sol";
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract ControlledGateway is AccessControl, Pausable {
+    bytes32 public constant GOVERNANCE_CONTROLLER =
+        keccak256("GOVERNANCE_CONTROLLER");
+
+    address public token;
+
+    constructor(address _token) {
+        _setupRole(DEFAULT_ADMIN_ROLE, _msgSender());
+        _setRoleAdmin(GOVERNANCE_CONTROLLER, DEFAULT_ADMIN_ROLE);
+
+        token = _token;
+    }
+
+    function addGovernanceController(address _newController)
+        external
+        whenNotPaused
+    {
+        grantRole(GOVERNANCE_CONTROLLER, _newController);
+    }
```

Comment:
can also use `Ownable` as a simpler alternative to `AccessControl`. The latter allows us to have specific roles for different categories of users which _might_ be useful (?), like in this particular case, pausing and unpausing can be controlled via governance. 

Curious to hear to your thoughts.

Reply:
In favour of access control. Also reinforced by https://github.com/livepeer/arbitrum-lpt-bridge/pull/11#discussion_r743197714